backend/db/models.py

```diff
@@ -168,3 +173,16 @@ class Extractor(TimestampedModel):
 
     def __repr__(self) -> str:
         return f"<Extractor(id={self.uuid}, description={self.description})>"
+
+
+def validate_extractor_owner(
+    session: Session, extractor_id: UUID, owner_id: UUID
+) -> Extractor:
+    """Validate the extractor id."""
+    extractor = (
+        session.query(Extractor).filter_by(uuid=extractor_id, owner_id=owner_id).first()
```

Comment:
.first() -> should usually be .scalar() or .one() if there are unique constraints